op-node/service.go

```diff
@@ -105,10 +105,11 @@ func NewConfig(ctx *cli.Context, log log.Logger) (*node.Config, error) {
 			Moniker: ctx.String(flags.HeartbeatMonikerFlag.Name),
 			URL:     ctx.String(flags.HeartbeatURLFlag.Name),
 		},
-		ConfigPersistence: configPersistence,
-		Sync:              *syncConfig,
-		RollupHalt:        haltOption,
-		RethDBPath:        ctx.String(flags.L1RethDBPath.Name),
+		ConfigPersistence:          configPersistence,
+		Sync:                       *syncConfig,
+		RollupHalt:                 haltOption,
+		RethDBPath:                 ctx.String(flags.L1RethDBPath.Name),
+		L1MaxConcurrentRPCRequests: ctx.Int(flags.L1RPCMaxConcurrentRequests.Name),
```

Comment:
This writes to the config attribute that shouldn't be there. Instead, load it into the endpoint config as part of the `NewL1EndpointConfig` function.